src/software/simulation/er_force_simulator.cpp

```diff
@@ -76,6 +75,52 @@ ErForceSimulator::ErForceSimulator(const TbotsProto::FieldType& field_type,
     this->resetCurrentTime();
 }
 
+/* returns a default realism config to simulate under ideal conditions */
+std::unique_ptr<RealismConfigErForce> ErForceSimulator::createDefaultRealismConfig()
```

Comment:
nit: Comments describing what a method does that the caller should also be aware of, should be added to the deceleration of the method in the header file.